src/main/migration/20220406061730-create-workout-exercise.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    return queryInterface.createTable('WorkoutExercise', {
```

Comment:
Running this migration is only necessary for the test db, NOT the dev db. Please let me know if there's a programmatic way to prevent this from being run on the dev db...

Reply:
You can probably have some sort of if statement that runs the migration only if `process.env.NODE_ENV === 'test'`